.github/helpers/pr-labels.yml

```diff
@@ -17,9 +17,6 @@
 - name: "ci:all-db"
   description: "Runs the builds for all databases."
   color: "cccccc"
-- name: "ci:authorizations"
```

Comment:
❓ Is there a reason related to this PR that the `ci:authorizations` is removed?

Reply:
Yes, we no longer need it for version 7.22-7.19 as the related stages has been merged to db-unit.